docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,489 @@
+    static_ham1 = 2 * np.pi * v1 * N1 + np.pi * anharm1 * N1 * (N1 - full_ident)
+    
+    static_ham_full = static_ham0 + static_ham1 + 2 * np.pi * J * ((a0 + a0dag) @ (a1 + a1dag))
+    
+    drive_op0 = 2 * np.pi * r0 * (a0 + a0dag)
+    drive_op1 = 2 * np.pi * r1 * (a1 + a1dag)
+
+Construct the :class:`.Solver` using the model details, including parameters necessary for pulse
+simulation. See the :class:`.Solver` documentation, as well as the :ref:`tutorial example <pulse
+solver>` for more details.
+
+.. jupyter-execute::
+
+    from qiskit_dynamics import Solver
+    
+    # build solver
+    dt = 1/4.5e9
+    
+    solver = Solver(
+        static_hamiltonian=static_ham_full,
+        hamiltonian_operators=[drive_op0, drive_op1, drive_op0, drive_op1],
+        rotating_frame=static_ham_full,
+        hamiltonian_channels=["d0", "d1", "u0", "u1"],
+        channel_carrier_freqs={"d0": v0, "d1": v1, "u0": v1, "u1": v0},
+        dt=dt,
+    )
+
+Next, instantiate the :class:`.DynamicsBackend`. The ``solver`` is used for simulation,
+``subsystem_dims`` indicates how the full system decomposes for measurement data computation, and
+``solver_options`` are consistent options used by :meth:`.Solver.solve` when simulating the
+differential equation. The full list of allowable ``solver_options`` are the arguments to
+:func:`.solve_ode`.
+
+Note that, to enable the internal automatic jit-compilation, we choose a JAX integration method.
+
+.. jupyter-execute::
+
+    from qiskit_dynamics import DynamicsBackend
+    
+    # Consistent solver option to use throughout notebook
+    solver_options = {"method": "jax_odeint", "atol": 1e-6, "rtol": 1e-8}
+    
+    backend = DynamicsBackend(
+        solver=solver,
+        subsystem_dims=[dim, dim], # for computing measurement data
+        solver_options=solver_options, # to be used every time run is called
+    )
+
+Alternatively to the above, the :meth:`.DynamicsBackend.from_backend` method can be used to build
+the :class:`.DynamicsBackend` from an existing backend. The above model, which was built manually,
+was taken from qubits :math:`0` and :math:`1` of ``almaden``.
+
+3. Simulating pulse schedules on the :class:`.DynamicsBackend`
+--------------------------------------------------------------
+
+With the above backend, we can already simulate a list of pulse schedules. The code below generates
+a list of schedules specifying experiments on qubit :math:`0`. The schedule is chosen to demonstrate
+that the usual instructions work on the :class:`.DynamicsBackend`.
+
+.. jupyter-execute::
+
+    %%time
+    
+    from qiskit import pulse
+    
+    sigma = 128
+    num_samples = 256
+    
+    schedules = []
+    
+    for amp in np.linspace(0., 1., 10):
+        gauss = pulse.library.Gaussian(
+            num_samples, amp, sigma, name="Parametric Gauss"
+        )
+    
+        with pulse.build() as schedule:
+            with pulse.align_right():
```

Comment:
this alignment is bit problematic because drive channel and measure channel are both ALAP scheduled in parallel. `align_sequential` should work in this example (because there is no parallel pulse).

Reply:
Changed it to `align_sequential`.